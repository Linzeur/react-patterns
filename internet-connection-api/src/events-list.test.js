import React from "react";
import { render } from "@testing-library/react";

import ListEvent from "./events-list";

const status = "Connected";
test("Test event list component", () => {
  const { asFragment } = render(
    <ListEvent title={"Titulo 1"} status={status} />
  );

  expect(asFragment()).toMatchSnapshot();
});
