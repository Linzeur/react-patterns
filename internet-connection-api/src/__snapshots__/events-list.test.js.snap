// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test event list component 1`] = `
<DocumentFragment>
  Titulo 1
  <ul>
    <li>
      Your state to internet is: Connected
    </li>
  </ul>
</DocumentFragment>
`;
